platform/linuxbsd/detect.py

```diff
@@ -86,9 +86,22 @@ def configure(env: "Environment"):
         env.Append(LINKFLAGS=["-rdynamic"])
 
     # CPU architecture flags.
-    if env["arch"] == "rv64":
+    if env["arch"] == "x86_64":
+        env.Append(CCFLAGS=["-m64", "-march=x86-64"])
+        env.Append(LINKFLAGS=["-m64", "-march=x86-64"])
+    elif env["arch"] == "x86_32":
+        env.Append(CCFLAGS=["-m32", "-march=i686"])
+        env.Append(LINKFLAGS=["-m32", "-march=i686"])
```

Comment:
Are both `-m64`/`-m32` and `-march` needed at the same time, or does the latter make the old `-m64` and `-m32` flags redundant?

Reply:
They are both required, yes.